internal/commands/vaultsecrets/secrets/create.go

```diff
@@ -130,11 +138,70 @@ type secretConfigInternal struct {
 }
 
 type SecretConfig struct {
-	Type            integrations.IntegrationType `hcl:"type"`
-	IntegrationName string                       `hcl:"integration_name"`
-	Details         cty.Value                    `hcl:"details"`
+	Type    integrations.IntegrationType `hcl:"type"`
+	Details cty.Value                    `hcl:"details"`
 }
 
+var (
+	twilioRotatingSecretConfig = map[string]any{
```

Comment:
Maybe calling these templates would be more appropriate since they are not meant to be modified